src/papyrus-lang-vscode/README.md

```diff
@@ -9,9 +9,11 @@
 </h1>
 <hr/>
 
-[![](https://vsmarketplacebadge.apphb.com/version-short/joelday.papyrus-lang-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=joelday.papyrus-lang-vscode)
-[![Build status](https://ci.appveyor.com/api/projects/status/ear84ovxacid2o1v?svg=true)](https://ci.appveyor.com/project/joelday/papyrus-lang)
-[![Discord](https://img.shields.io/discord/558746231665328139.svg?color=%23738ADB)](https://discord.gg/E4dWujQ)
+[![](https://vsmarketplacebadge.apphb.com/version-short/joelday.papyrus-lang-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=joelday.papyrus-lang-vscode) [![Build status](https://ci.appveyor.com/api/projects/status/ear84ovxacid2o1v?svg=true)](https://ci.appveyor.com/project/joelday/papyrus-lang) [![Discord](https://img.shields.io/discord/558746231665328139.svg?color=%23738ADB)](https://discord.gg/E4dWujQ)
```

Comment:
I think the appveyor link is actually including builds running on non master branches. Can you fix that up, too?

Reply:
ok, done